FRAuth/FRAuth/Device/HardwareCollector.swift

```diff
@@ -18,6 +18,9 @@ public class HardwareCollector: DeviceCollector {
     /// Name of current collector
     public var name: String = "hardware"
     
+    /// Initializes BrowserCollector instance
```

Comment:
I think you meant `HardwareCollector`, rather than `BrowserCollector` :)

Reply:
Thanks Justin, I've updated the inline comment. :)